connector/rest/src/main/java/it/pagopa/selfcare/pagopa/backoffice/connector/rest/client/GecRestClient.java

```diff
@@ -0,0 +1,40 @@
+package it.pagopa.selfcare.pagopa.backoffice.connector.rest.client;
+
+import it.pagopa.selfcare.pagopa.backoffice.connector.api.GecConnector;
+import it.pagopa.selfcare.pagopa.backoffice.connector.model.gec.BundleType;
+import it.pagopa.selfcare.pagopa.backoffice.connector.model.gec.Bundles;
+import it.pagopa.selfcare.pagopa.backoffice.connector.model.gec.Touchpoints;
+import org.springframework.cloud.openfeign.FeignClient;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestHeader;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import java.util.ArrayList;
+
+@FeignClient(name = "${rest-client.gec.serviceCode}", url = "${rest-client.gec.base-url}")
+public interface GecRestClient extends GecConnector {
+
+    @GetMapping(value = "${rest-client.gec.getBundlesByCI.path}", produces = MediaType.APPLICATION_JSON_VALUE)
+    @ResponseBody
+    Bundles getBundlesByCI(@RequestParam(required = true) String cifiscalcode,
+                           @RequestParam(required = false) Integer limit,
+                           @RequestParam(required = false) Integer page,
+                           @RequestHeader(name = "X-Request-Id", required = false) String xRequestId);
```

Comment:
You can use a FeignConfiguration to add common headers (for ex: `requestID` or `apikey`) to every call.

Reply:
we can use FeignConfiguration in future. It's an impactful change